tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -531,4 +531,52 @@ describe('Test application deployment based on clusterGroup', { tags: '@p1'}, ()
       )
     })
   )
+
+  qase(26,
+    it("Fleet-26: Test install single application to the all defined clusters in the 'clusterGroup'", { tags: '@fleet-26' }, () => {
+      const repoName = 'default-single-app-cluster-group-26'
+      const path2 = 'multiple-paths/config'
+
+      cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+      cy.contains('.title', 'Clusters').should('be.visible');
+
+      // Assign label to the clusters 
+      dsClusterList.forEach(
+        (dsClusterName) => {
```

Comment:
Isn't  `dsClusterName` pointing solely to `imported-0`?
If so, isn't the idea that is able to iterate over several ds clusters but pick solely the one expected?

Reply:
It is going to iterate over all the clusters available in `dsClusterList`.

Reply:
Ah, true; it is declared earlier. I got confused given that `dsClusterName` is also defined as `imported-0` only globally. In this regard, if you consider it, perhaps it would not be bad to rename `dsClusterName`in order to be distinguished from the general one.

Reply:
Thanks for the suggestion, I will rename `dsClusterName`, when I re-factor all clusterGroup test cases.

Reply:
Perfect. Thanks